File: app/src/main/java/com/example/capstona_a/data/AvgStats.java
package com.example.capstona_a.data;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AvgStats {

    @SerializedName("kills")
    @Expose
    private Double kills;
    @SerializedName("deaths")
    @Expose
    private Double deaths;
    @SerializedName("assists")
    @Expose
    private Double assists;
    @SerializedName("gold")
    @Expose
    private Double gold;
    @SerializedName("damage_dealt")
    @Expose
    private Double damageDealt;
    @SerializedName("damage_taken")
    @Expose
    private Double damageTaken;
    @SerializedName("vision")
    @Expose
    private Double vision;
    @SerializedName("exp")
    @Expose
    private Double exp;


}

File: app/src/main/java/com/example/capstona_a/data/AvgStats.kt
package com.example.capstona_a.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class AvgStats(

    @SerializedName("kills")
    @Expose
    val kills: Double? = null,

    @SerializedName("deaths")
    @Expose
    val deaths: Double? = null,

    @SerializedName("assists")
    @Expose
    val assists: Double? = null,

    @SerializedName("gold")
    @Expose
    val gold: Double? = null,

    @SerializedName("damage_dealt")
    @Expose
    val damageDealt: Double? = null,

    @SerializedName("damage_taken")
    @Expose
    val damageTaken: Double? = null,

    @SerializedName("vision")
    @Expose
    val vision: Double? = null,

    @SerializedName("exp")
    @Expose
    val exp: Double? = null,
)